Fixtures/ClangModules/CBlocks/Sources/clib/foo.c

```diff
@@ -0,0 +1,5 @@
+#include "foo.h"
+
+int operate(int a, int b, Operation operation) {
+    return operation(a,b);
+}
```

Comment:
Let's have the C code also create a block, which I think exercises a few more entry points.